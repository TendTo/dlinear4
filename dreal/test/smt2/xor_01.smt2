(declare-const x Real)
(declare-const y Real)
(assert (xor (>= x 10) (>= y -10)))
(check-sat)
(get-model)
(exit)
