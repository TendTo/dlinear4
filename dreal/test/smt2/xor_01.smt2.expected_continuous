delta-sat with delta = 5e-324 ( > 0)
(model
  (define-fun x () Real 10)
  (define-fun y () Real -10)
)
