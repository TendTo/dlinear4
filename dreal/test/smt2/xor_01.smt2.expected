delta-sat with delta = 0.001
(model
  (define-fun x () Real 10)
  (define-fun y () Real -10)
)
